concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -1743,7 +1741,7 @@ jobs:
                   UAA_CLIENT_SECRET=$($VAL_FROM_YAML secrets.uaa_admin_client_secret cf-secrets/cf-secrets.yml)
                   cd paas-cf/scripts
                   bundle
-                  bundle exec sync-admin-users.rb ${API_ENDPOINT} ../config/admin_users.yml "{{NEW_ACCOUNT_EMAIL_ADDRESS}}"
+                  bundle exec sync-admin-users.rb ${API_ENDPOINT} ../config/admin_users.yml "((disable_user_creation))"
```

Comment:
I think we should leave the NEW_ACCOUNT_EMAIL_ADDRESS value here, as the script doesn't handle the "true" value. If the NEW_ACCOUNT_EMAIL_ADDRESS is not set it will throw an error as probably intended.